README.md

```diff
@@ -456,12 +458,44 @@ spring.data.mongodb.uri=mongodb://${embedded.mongodb.host}:${embedded.mongodb.po
 * embedded.google.pubsub.project-id
 
 ##### Example
-To auto-configure spring-cloud-gcp-starter-pubsub use these properties in your test application.properties:
+To auto-configure spring-cloud-gcp-starter-pubsub use these properties in your test `application.properties`:
 ```
 spring.cloud.gcp.project-id=${embedded.google.pubsub.project-id}
 spring.cloud.gcp.pubsub.emulatorHost=${embedded.google.pubsub.host}:${embedded.google.pubsub.port}
 ```
 
+### embedded-keycloak
+
+##### Maven dependency
+```xml
+<dependency>
+    <groupId>com.playtika.testcontainers</groupId>
+    <artifactId>embedded-keycloak</artifactId>
+    <scope>test</scope>
+</dependency>
+```
+
+##### Consumes (via bootstrap.properties)
+* `embedded.keycloak.enabled` (boolean, true|false, default is 'true')
+* `embedded.keycloak.dockerImage` (String, default is set to 'jboss/keycloak:8.0.1')
+  * You can pick wanted version on [dockerhub](https://hub.docker.com/r/jboss/keycloak)
+* `embedded.keycloak.command` (String[], default is '["-c", "standalone.xml", "-Dkeycloak.profile.feature.upload_scripts=enabled"]')
```

Comment:
Sorry, I was asking for extracting not the whole command, only `standalone.xml`

Reply:
Yes but I think this is more flexible tough. Why not making the whole command configurable?

Reply:
Okay